gke/README.md

```diff
@@ -2,8 +2,8 @@
 
 ## Assumptions and Requirements
 
-It is assumed that a Google Cloud project exists and that the deployer
-has permissions to create, modify, and delete resources and Service Accounts.
+It is assumed that a Google Cloud project exists and that the Compute Engine Default Service Account has the following roles: 
+Role Administrator, Security Admin, Project IAM Admin.
```

Comment:
Where are these roles coming from and why are you assuming that the user will use the Compute Engine Default Service Account?